### ISSUE TITLE
Static router produce double slash

### ISSUE THREAD

**Issue by diondirza** [ReactTraining/react-router#4480]
## Version
4.0.0-beta.5

## Test Case
Use it on isomorphic react app

## Steps to reproduce
using `StaticRouter` without basename props
`<StaticRouter location={request.url} context={routerContext}>`
and have `Link` with prop to `/home`
`<Link to={'/home'}>`

## Expected Behavior
Render result:
`<a href="/home' />`

## Actual Behavior
Render result:
`<a href="//home" />`

I got this warning from my client console when I upgrade to latest beta,
```
Warning: React attempted to reuse markup in a container but the checksum was invalid. This generally means that you are using server rendering and the markup generated on the server was not what the client was expecting. React injected new markup to compensate which works but you have lost many of the benefits of server rendering. Instead, figure out why the markup being generated is different on the client or server:
 (client) href="/home" data-reactid="21
 (server) href="//home" data-reactid="21
```
and lead me to find this [culprit](https://github.com/ReactTraining/react-router/blob/v4/packages/react-router/modules/StaticRouter.js#L80).

**Comment by PEM--** [comment]
Confirmed for me as well.

**Comment by pshrmn** [comment]
Fixed in #4484

**Comment by bryamrrr** [comment]
I have this exact problema using react-router-dom
